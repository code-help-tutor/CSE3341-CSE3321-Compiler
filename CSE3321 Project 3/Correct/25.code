procedure c25 is
	array x;
	array y;
begin
	array n;
	x := new integer[1];
	n := new integer[1];
	n := 5;
	x := 4;
	y := array x;
	while not x = 0 do
		n := n*x;
		y := y-1;
		out(n);
	end
end