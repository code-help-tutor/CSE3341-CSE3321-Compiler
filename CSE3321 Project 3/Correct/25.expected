20
60
120
120
